src/extension/geofence/geofence/src/main/resources/applicationContext.xml

```diff
@@ -93,7 +99,18 @@
             <constructor-arg index="0" ref="${ruleReaderFrontend}" />
             <constructor-arg index="1" ref="rawCatalog" />
             <constructor-arg index="2" ref="geofenceConfigurationManager" />
-            <constructor-arg index="3" ref="geoFenceWpsHelper" />
+            <constructor-arg index="3" >
+                <!-- Resolver without cache -->                
+                <!--                
+                <bean class="org.geoserver.geofence.containers.ContainerAccessResolver">
+                    <constructor-arg ref="${ruleReaderFrontend}"/>
+                </bean>-->                
```

Comment:
Left for future debugging purposes? If so, please write explicitly in the comment.